cmd/recoverwallet.go

```diff
@@ -54,7 +72,8 @@ var recoverwalletcmd = &cobra.Command{
 }
 
 func init() {
-	rootCmd.AddCommand(recoverwalletcmd)
+	rootCmd.AddCommand(WithoutZCNCore(WithoutWallet(recoverwalletcmd)))
 	recoverwalletcmd.PersistentFlags().String("mnemonic", "", "mnemonic")
+	recoverwalletcmd.PersistentFlags().Bool("offline", false, "recover wallet without registration on blockchain")
```

Comment:
Will the recovered wallet be registered to 0chain the next time when it is used by other online command? if so, perhaps we can set this as an offline command by default.

Reply:
@peterlimg A wallet recovered by seed phrase using `--offline` needs to be registered using `register` command while connected to network. `getbalance` does not register it for example.

Reply:
@peterlimg  `recoverwallet` means recover wallet from mnemonic, and save it into ~/.zcn/wallet.json or --wallet. `registration` means to register/active it to miners again. so they are used in different scenes. I would suggest to leave it as it is. Because windows app is working with it. it will be broken if default behavior is changed.

Reply:
okay, then let's keep the current logic